### PR TITLE
implement the Aqua-park validate logic

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -2,24 +2,77 @@
...
+        self.private_attribute = f"_{name}"
+
+    def __get__(self, instance: object, owner: object) -> bool:
+        return getattr(instance, self.private_attribute)
+
+    def __set__(self, instance: object, value: int) -> None:
+        if self.max_amount >= value >= self.min_amount:
+            setattr(instance, self.private_attribute, True)
+        else:
+            setattr(instance, self.private_attribute, False)
```
Comment (ArsenPidhoretskyi) [inline app/main.py]: You should set original value, in other case raise an exception

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (vasylhnatiuk) [reply app/main.py]: do it

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]
you pushed nothing

File: app/main.py [hunk 2]
```diff
@@ -2,24 +2,75 @@
...
+    height = IntegerRange(120, 220)
+    weight = IntegerRange(50, 120)
 
 
 class Slide:
-    pass
+
+    def __init__(self,
+                 name: str,
+                 limitation_class: type(SlideLimitationValidator)
```
Comment (ArsenPidhoretskyi) [inline app/main.py]:
```suggestion
                 limitation_class: type[SlideLimitationValidator]
```

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
No need to call type() use type[] for annotations

**Review by vsmutok — APPROVED** [review]
GJ!

**Pull request closed without merging**